verification_token/models.py

```diff
@@ -95,6 +103,12 @@ def check_key(self, key):
         """
         return self.is_valid and self.key == key
 
+    def set_extra_data(self, extra_data):
+        self.extra_data = pickle.dumps(extra_data)
+
+    def get_extra_data(self):
+        return pickle.loads(self.extra_data)
```

Comment:
ještě bych se zamyslel nad tím zda opravdu chceme povolit ukládat všechno možné včetně class atp. Jestli by nestačilo serializovat jsonem

Reply:
obecně moc pickle nemám rád, jelikož to není úplně bezpečné

Reply:
JSON asi bude stačit, ale nechtěl jsem to omezovat, tím pádem budou povoleny pouze primitivní Python objekty asi, že?